processor/README.md

```diff
@@ -244,6 +245,37 @@ span:
     separator: <value>
 ```
 
+In the second form it takes a list of regular expressions to match span name against
+and extract attributes from based on subexpressions.
+
+`rules` is a list of rules to extract attribute values from span name. The values
+in the span name are replaced by extracted attribute names. Each rule in the list
+is regex pattern string. Span name is checked against the regex and if the regex
+matches then all named subexpressions of the regex are extracted as attributes
+and are added to the span. Each subexpression name becomes an attribute name and
+subexpression matched portion becomes the attribute value. The matched portion
+in the snap name is replaced by extracted attribute name. If the attributes
+already exist in the span then they will be overwritten. The process is repeated
```

Comment:
Is it too costly to do like `insert|upsert` with `upsert` being the default?

Reply:
Can you please clarify?

Reply:
Just considering in the case if the attribute key already exists. The attribute processor handles that by having distinct actions: `update|insert|upsert`, the case that matters here is if the same attribute key is already present. I agree that the default behavior should be `upsert`, just considering if is worth to have a `insert` (ie.: if the attribute key already exists it keeps the original).

Reply:
I understand. We can add the action later if needed and make current behavior the default.